modules/mono/utils/mono_reg_utils.cpp

```diff
@@ -58,10 +58,10 @@ REGSAM _get_bitness_sam() {
 }
 
 LONG _RegOpenKey(HKEY hKey, LPCWSTR lpSubKey, PHKEY phkResult) {
-	LONG res = RegOpenKeyExW(hKey, lpSubKey, 0, Key::READ, phkResult);
+	LONG res = RegOpenKeyExW(hKey, lpSubKey, 0, KEY_READ, phkResult);
 
 	if (res != ERROR_SUCCESS)
-		res = RegOpenKeyExW(hKey, lpSubKey, 0, Key::READ | _get_bitness_sam(), phkResult);
+		res = RegOpenKeyExW(hKey, lpSubKey, 0, KEY_READ| _get_bitness_sam(), phkResult);
```

Comment:
There's a style issue, please amend the commit to fix it:
```suggestion
		res = RegOpenKeyExW(hKey, lpSubKey, 0, KEY_READ | _get_bitness_sam(), phkResult);
```

While at it, making the commit message a tad more explicit would be appreciated. E.g.:
```
Fix mono build after invalid rename of `KEY_READ`

Follow-up to #51684.
```

Reply:
Pushed an amend with a fix myself so we can fix the Mono+Windows build.